tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml

```diff
@@ -351,108 +355,138 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
-		TSN specific function(s) of Edge Nodes (T-PE) are belonging to the 
-		native service processing (NSP) <xref target="RFC3985"/> 
-		block. This is similar to other functionalities being defined by standard 
-		bodies other than the IETF (for example in case of Ethernet: stripping, 
-		overwriting or adding VLAN tags, etc.). Depending on the TSN role of 
-		the Edge Node in the end-to-end TSN service selected TSN functions 
-		must be supported. 
+                replication and elimination functions for a TSN network.
+          </t>
+          <t>
+                TSN specific functions are executed on the data received by 
+                the PE from the CE before presentation to the DetNet PW for 
+                transmission across the DetNet domain, or on the data received from 
+                a DetNet PW by a PE before it is output on the Attachment Circuit (AC). 
+          </t>
+          <t>
+                TSN specific function(s) of Edge Nodes (T-PE) are belonging to the 
+                native service processing (NSP) <xref target="RFC3985"/> 
+                block. This is similar to other functionalities being defined by standard 
+                bodies other than the IETF (for example in case of Ethernet: stripping, 
+                overwriting or adding VLAN tags, etc.). Depending on the TSN role of 
+                the Edge Node in the end-to-end TSN service selected TSN functions 
+                must be supported. 
     </t>
     <t>
-		Implementations of this document SHALL use management and
-		control information to ensure TSN specific functions of the Edge Node
-		according to the expectations of the connected TSN network. 
+                Implementations of this document SHALL use management and
+                control information to ensure TSN specific functions of the Edge Node
+                according to the expectations of the connected TSN network. 
     </t>
   </section>
 
   <section title="Edge Node DetNet Service Proxy Procedures" anchor="tom_svc_prx_proc">
-  	<t>
-		The Service Proxy function maps between App-flows and DetNet flows. 
-		It MUST support the TSN Stream identification functions and the 
-		related managed objects as defined in 
-		<xref target="IEEE8021CB">IEEE 802.1CB</xref> and 
-		<xref target="IEEEP8021CBdb">IEEE P802.1CBdb</xref> to recognize 
-		the App-flow related packets. 
+          <t>
+                The DetNet Edge Node TSN function MUST support the TSN Stream
+                identification functions and the related managed objects as
+                defined in <xref target="IEEE8021CB">IEEE 802.1CB</xref> and
+                <xref target="IEEEP8021CBdb">IEEE P802.1CBdb</xref> to
+                recognize the App-flow related packets.  The service Proxy
+                presents TSN flows as a single application to a DetNet Flow.
+                The DetNet Flow is a statically defined service that connects
+                TSN service proxy capable Nodes. 

```

Comment:
Delete new sentences here. What is "The DetNet Flow is a statically defined service" ??? We have definition for DetNet flow.